packages/edit-site/src/components/sidebar/index.js

```diff
@@ -1,20 +1,35 @@
 /**
  * WordPress dependencies
  */
-import { createSlotFill, Panel } from '@wordpress/components';
+import { createSlotFill } from '@wordpress/components';
+import { PluginComplementaryArea } from '@wordpress/interface';
 import { __ } from '@wordpress/i18n';
+import { cog, pencil } from '@wordpress/icons';
 
 const { Slot: InspectorSlot, Fill: InspectorFill } = createSlotFill(
 	'EditSiteSidebarInspector'
 );
-
 function Sidebar() {
 	return (
-		<div className="edit-site-sidebar">
-			<Panel header={ __( 'Inspector' ) }>
+		<>
+			<PluginComplementaryArea.Slot scope="edit-site" />
+			<PluginComplementaryArea
+				scope="edit-site"
+				complementaryAreaName="edit-site/block-inspector"
```

Comment:
Would it work if we would simplify it a bit to:
```suggestion
				name="block-inspector"
```

`complementaryAreaName` could be combined as `scope` + `name`. What do you think?

Reply:
In most cases, the name of the area is `${ context.name }/${ ownProps.name }`.
That means an area is named with the plugin id of the plugin that originated it and the name of the sidebar.
I'm not sure if we should include the scope as the name of the area, because in the store the area is already in a tree under the scope.

In the case for our own core sidebars, we are passing an explicit name I totally agree the name could be "block-inspector" instead of "edit-site/block-inspector", I used the later because in edit-post we were also using "edit-post/document" instead of just "document" and "edit-post/block" instead of "block". Renaming things in this PR will make it even bigger, as we would need back-compatibility for the previous names.